camel-core/src/main/java/org/apache/camel/model/ProcessorDefinition.java

```diff
@@ -568,11 +543,328 @@ public Type attribute(QName name, Object value) {
      * @return the builder
      */
     @SuppressWarnings("unchecked")
+    public Type fork(String uri) {
+        addOutput(new ForkDefinition(uri));
+        return (Type) this;
+    }
+
+    /**
+     * Sends the exchange to the given endpoint
+     *
+     * @param uri  the String formatted endpoint uri to send to
+     * @param args arguments for the string formatting of the uri
+     * @return the builder
+     */
+    @SuppressWarnings("unchecked")
+    public Type fork(String uri, Object... args) {
```

Comment:
This was forkF( in congruence with toF - F for format ), not sure why you've made it fork again.